packages/client/src/screens/settings/settings.screen.tsx

```diff
@@ -95,3 +140,16 @@ const MySettingsScreen = () => {
 export const SettingsScreen = withNavigation(MySettingsScreen, {
   title: 'Settings',
 });
+
+export const ColorDivs = () => {
```

Comment:
There needs to be a better option to load in tailwind classes... I couldn't figure it out. :)

Reply:
Found a fix (tailwind safelist)